spec/grape/validations/contract_scope_spec.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'pry'
+
+describe Grape::Validations::ContractScope do
```

Comment:
Does it make sense to extract a shared spec that applies to any scope and say "behaves like a scope" here?

Reply:
Maybe not.

`ParamsScope` tests a lot of functionality that we don't test here because it's part of the third-party package (`dry-schema`). As a bonus, we also test here being able to combine both `contract` and `params`.